migrations/20240906-add-templateId-and-archive-index-to-builds.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable new-cap */
+
+'use strict';
+
+const prefix = process.env.DATASTORE_SEQUELIZE_PREFIX || '';
+const table = `${prefix}jobs`;
+
+module.exports = {
+    // eslint-disable-next-line no-unused-vars
+    up: async (queryInterface, Sequelize) => {
+        await queryInterface.sequelize.transaction(async transaction => {
+            await queryInterface.removeIndex(table, `${table}_templateId`, { transaction });
+
+            await queryInterface.addIndex(table, {
+                name: `${table}_templateId_archived`,
+                fields: ['templateId', 'archived'],
+                transaction
+            });
+        });
+    },
+    // eslint-disable-next-line no-unused-vars
+    down: async (queryInterface, Sequelize) => {
+        await queryInterface.sequelize.transaction(async transaction => {
+            await queryInterface.removeIndex(table, `${table}_templateId_archived`, { transaction });
+
+            await queryInterface.addIndex(table, {
+                name: `${table}_templateId`,
+                fields: ['templateId'],
+                transaction
+            });
+        });
+    }
```

Comment:
is this right?

Reply:
@tkyi
I have confirmed locally that it works fine.
I have added it for easy rollback, but I am aware that it was not in the previous migration files.
If it is not needed, we will remove it. What do you think?

Reply:
Not being absolutely necessary, I removed them according to customary practice.

Reply:
Ah I see thanks for the context